### DISCUSSION TITLE
Flooz Wallet rejected even though it does work (Project ID: 1693efc9e115ee372b8f182ec6557b7a)

### DISCUSSION THREAD

**Discussion by plmmtt** [WalletConnect/walletconnect-monorepo#2779] [Developer Support]
I am submitting a wallet app to the WalletConnect explorer but it gets rejected with the following reason: 

Reason: Does not support Sign V2. Docs to help migrate to Sign V2: https://docs.walletconnect.com/2.0/advanced/migration-from-v1.x/wallets/wallet-checklist Please also test your integration signs successfully using our test dapp: https://lab.web3modal.com/with-sign-api/html 

When I try to test the integration using the provided link the link brings me to a 404 page, when I test it using [this link](https://react-app.walletconnect.com/) instead it works flawlessly.

**Comment by crypblizz8** [comment]
@JeroldG

**Reply by JeroldG** [reply]
I have tested and now approved your listing!
